src/stories/Quiz/Hooks/index.tsx

```diff
@@ -0,0 +1,344 @@
+/* eslint-disable jsx-a11y/alt-text */
+/* eslint-disable react/button-has-type */
+import { QuestionOption } from '@constructor-io/constructorio-client-javascript/lib/types';
+import React from 'react';
+import RedoSVG from '../../../components/RedoButton/RedoSVG';
+import { useCioQuiz } from '../../../index';
+import '../../../styles.css';
+import { getStoryParams } from '../../../utils';
+
+export default function HooksTemplate(args) {
```

Comment:
This shouldn't look like this, right?

![image](https://github.com/Constructor-io/constructorio-ui-quizzes/assets/2954126/c1be2327-7c94-431f-ad7f-ec6593d927f4)